html_test_harness/domain_store.py

```diff
@@ -0,0 +1,85 @@
+import csv
+from pkg_resources import resource_filename
+
+from diskcache import Index
```

Comment:
This package is pretty amazing. It transparently pickles things to the disk for you so we can store the actual Response objects coming back from requests and inspect them.